disnake/ext/commands/slash_core.py

```diff
@@ -299,6 +327,29 @@ def __init__(
                 "Cannot set `default_member_permissions` or `dm_permission` on subcommands"
             )
 
+    @property
+    def root_parent(self) -> InvokableSlashCommand:
+        """:class:`InvokableSlashCommand`: Returns the slash command containing this subcommand,
```

Comment:
```suggestion
        """:class:`InvokableSlashCommand`: Returns the top-level slash command containing this subcommand,
```
might be good clarification

Reply:
https://github.com/DisnakeDev/disnake/pull/759/commits/28f3177cd9f54eca62b5d866a162e800c6406b96